charts/pokt-portal-v2/templates/deployment.yaml

```diff
@@ -73,19 +73,24 @@ spec:
             successThreshold: {{ .Values.middleware.startupProbe.successThreshold | default 1 }}
           {{- end }}
           volumeMounts:
-          {{- if .Values.externalSecrets.enabled }}
-          - mountPath: "/config/"
-            name: external-secrets
-            readOnly: true
-          {{- else }}
-          - mountPath: /config
+          - mountPath: {{ .Values.middleware.configMountPath | default "/config" }}
             name: config
-          {{- end }}
           env:
             - name: CONFIG_FILE
-              value: "/config/config.json"
+              value: {{ printf "%s/%s" ( trimSuffix "/" .Values.middleware.configMountPath ) "config.json"}}
             - name: PORT
-              value: "{{ .Values.service.port | default "8080" }}"
+              value: "{{ .Values.middleware.containerPort | default 80 }}"
+            - name: GATEWAY_PRIVATE_KEY
```

Comment:
The rest of these env vars should be moved into config.json so we at least have some consistency in where we are pulling them from. Requires code change on portal-middleware